### DISCUSSION TITLE
How termux is working on Android 11+?

### DISCUSSION THREAD

**Discussion by subham008** [termux/termux-app#4147] [Q&A]
How is termux executing executable on Android 11 and above 
when Android restricted execution of executable from even app private directory or internal storage???

**Comment by agnostic-apollo** [comment]
It uses `targetSdkVersion=28`.

https://github.com/agnostic-apollo/Android-Docs/blob/master/site/pages/en/projects/docs/apps/processes/app-data-file-execute-restrictions.md

**Reply by subham008** [reply]
On play store in termux description it is written "Required OS: Android 11 and up"
So I don't think so it's target SDK is 28 if it they wouldn't be able to update termux in play store

**Reply by agnostic-apollo** [reply]
Playstore one is using `system_linker_exec` hack. Its documented in the docs.
